tests/tests.py

```diff
@@ -619,6 +623,27 @@ def test_add_and_remove_user(self):
         result = self.jira.delete_user(self.test_username)
         assert result, True
 
+    def test_create_team(self):
+        if self._should_skip_for_pycontribs_instance():
+            self._skip_pycontribs_instance()
```

Comment:
We use a test decorator to enable the tests running on the Jira Cloud instance, this is denoted by `@allow_on_cloud`, an example is in `test_group`